README.md

```diff
@@ -134,12 +134,18 @@ You enable caching if you pass a configured Cache Pool Object to the static cons
 $cachePool = new ArrayCachePool();
 
 // Pass it as a 2nd argument
-\OpenAPIValidation\PSR7\ResponseValidator::fromYamlFile($yamlFile, $cachePool);
+$validator = (new \OpenAPIValidation\PSR7\ValidatorBuilder)
+    ->fromYamlFile($yamlFile)
+    ->setCache($cachePool)
+    ->getResponseValidator();
 # or
 \OpenAPIValidation\PSR15\ValidationMiddleware::fromYamlFile($yamlFile, $cachePool);
 ```
 
-Cache does not set expiration date for its payload.
+You can use `->setCache($pool, $ttl)` call for both PSR-7 and PSR-15 builder in order to set proper ttl (or explicit `null`)
```

Comment:
Please make it clear that `null` means no expiration at all, so cache item will be stored for as long as cache implementation allows.

Reply:
I do not. `null` means PSR-6 terms null, which means `default to the driver`, which could be anything

```
If a calling library asks for an item to be saved but does not specify an expiration time, or specifies a null expiration time or TTL, an Implementing Library MAY use a configured default duration. If no default duration has been set, the Implementing Library MUST interpret that as a request to cache the item forever, or for as long as the underlying implementation supports.
```

I'm not sure we should re-print PSR-6 standard here. But maybe link to it

Reply:
Makes sense. My intention was just to give a clear note to the developer what `null` means.

Reply:
Add a link, check it